.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Wasn't this a part of a different PR? 🤓

Reply:
yes, sorry about this, must have messed up somewhere when merging branches

Reply:
All good!